src/views/PeopleSearch/components/SearchFieldList/index.tsx

```diff
@@ -521,28 +524,66 @@ const SearchFieldList = ({ onSearch }: Props) => {
                       select
                     />
                   ) : (
-                    <Grid item width={225}>
-                      <Slider
-                        getAriaLabel={() => 'graduationYearRange'}
-                        value={graduationYearRange}
-                        onChange={handleSliderChange}
-                        valueLabelDisplay="auto"
-                        getAriaValueText={toString}
-                        min={1889}
-                        max={currentYear}
-                        disabled={!searchParams.includeAlumni}
-                        color="secondary"
-                      />
-                      <Typography fontSize={15} align="center">
-                        {graduationYearRange[0]}-{graduationYearRange[1]}
-                      </Typography>
+                    <Grid item container spacing={1}>
+                      <Grid item xs={6} md={6.75}>
+                        <SearchField
+                          name="initial_year"
+                          value={searchParams.initial_year}
+                          updateValue={handleUpdate}
+                          options={Array.from(
+                            {
+                              length:
+                                Number(searchParams.final_year) > 0
+                                  ? Number(searchParams.final_year) - 1889 + 1
+                                  : currentYear - 1889 + 1,
```

Comment:
```suggestion
                                searchParams.final_year
                                  ? Number(searchParams.final_year) - 1889 + 1
                                  : currentYear - 1889 + 1,
```

Javascript has falsey zero (0 == false, all other numbers == true), my personal preference is that we utilize this. (This also applies to all the uses of  `Number(searchParams.final_year) > 0` found below